src/css/components/_statistic.scss

```diff
@@ -20,64 +21,35 @@ il-statistic,
   --il-statistic-text-color: var(--il-heading-color);
   --il-statistic-heading-color: var(--il-heading-color);
 
-  &[size="small"] {
-    --il-statistic-font-size: 1.125rem;
-    --il-statistic-line-height: 1.188rem;
-  }
-  &[size="large"] {
-    --il-statistic-font-size: 1.375rem;
-    --il-statistic-line-height: 1.625rem;
-  }
-  &[size="extralarge"] {
-    --il-statistic-font-size: 1.5rem;
-    --il-statistic-line-height: 1.75rem;
-  }
   em {
     font-style: normal;
   }
+}
 
-  &.il-orange {
-    --il-statistic-heading-color: var(--il-orange);
-  }
-
-  &.il-industrial-blue {
-    --il-statistic-heading-color: var(--il-industrial-blue);
-  }
-
-  &.il-arches-blue {
-    --il-statistic-heading-color: var(--il-arches-blue);
-  }
-
-  &.il-arches-blue-dark {
-    --il-statistic-heading-color: var(--il-arches-blue-darker-2);
-  }
-
-  &.il-patina {
-    --il-statistic-heading-color: var(--il-patina-darker-1);
-  }
+.il-size-small il-statistic, il-statistic.il-size-small {
+  --il-statistic-font-size: 1.125rem;
+  --il-statistic-line-height: 1.188rem;
+  --il-statistic-stat-font-size: 3rem;
+  --il-statistic-stat-padding: 13px 0 20px 0;
+  --il-statistic-source-font-size: 1rem;
+  --il-statistic-source-line-height: 1.25rem;
+}
 
-  &.il-harvest {
-    --il-statistic-heading-color: var(--il-harvest-lighter-2);
-    --il-statistic-text-color: var(--il-harvest-lighter-2);
-  }
 
-  &.il-harvest-light {
-    --il-statistic-heading-color: var(--il-harvest-lighter-4);
-    --il-statistic-text-color: var(--il-harvest-lighter-4);
-  }
+.il-size-large il-statistic, il-statistic.il-size-large {
+  --il-statistic-font-size: 1.375rem;
+  --il-statistic-line-height: 1.625rem;
+  --il-statistic-stat-font-size: 5.875rem;
+  --il-statistic-stat-padding: 30px 0 35px 0;
+  --il-statistic-source-font-size: 1.125rem;
+}
 
-  &.il-orange-light {
-    --il-statistic-heading-color: var(--il-orange-lighter-2);
-    --il-statistic-text-color: var(--il-orange-lighter-2);
-  }
-  
-  &.il-arches-blue-light {
-    --il-statistic-heading-color: var(--il-arches-blue-lighter-2);
-    --il-statistic-text-color: var(--il-arches-blue-lighter-2);
-  }
-    
-  &.il-blue-dark {
-    --il-statistic-heading-color: var(--il-blue-darker-1);
-    --il-statistic-text-color: var(--il-blue-darker-1);
-  }
+.il-size-extralarge il-statistic, il-statistic.il-size-extralarge {
```

Comment:
Let's use xlarge instead of extralarge

Reply:
I changed it to "xlarge" -- wasn't sure if we should use "xlarge" or "x-large". I do like "il-size-xlarge" better than "il-size-x-large"